pyproject.toml

```diff
@@ -18,31 +44,53 @@ exclude = '''
     | .venv
     | .build
     | build
+    | lib
     | report
+    | docs
 )/
 '''
 
 [tool.isort]
 profile = "black"
-skip_glob = [
-  ".eggs",
-  ".git",
-  ".tox",
-  ".venv",
-  ".build",
-  "build",
-  "report"
-]
+line_length = 99
+skip_glob = [".eggs", ".git", ".tox", ".venv", ".build", "build", "lib", "report"]
 
 [tool.pylint]
 max-line-length = 99
-extension-pkg-allow-list = "pygit2"
```

Comment:
test failing because this was not allowed?

Reply:
Yep I think it was, although those workflows and scripts were moved to https://github.com/canonical/snap-tempest-automation , so they shouldn't be in any branch in this repo - looks like they've already been deleted from the other branches, but this branch was missed. I've removed the files in this branch now.